tests/basics/BuiltinsTest.py

```diff
@@ -524,3 +524,32 @@ def __complex__(self):
     print("Oops, must not happen.")
 else:
     print("Oops, must not happen.")
+
+# Test for any builtins
+print(any([None, None, None]))
+print(any([None, 4, None]))
+print(any([]))
+print(any(range(20)))
+print(any([0] * 20000))
+print(any([0] * 255))
+
+S = [40, 60, 30]
+print(any(x > 42 for x in S))
```

Comment:
Can you make it so that we can see if and when any stops iteration, i.e. could you make that iterator some that does an output.

```
def S():
    print("Yielding 40")
    yield 40
    print("Yielding 60")
    yield 60
    print("Yielding 30")
    yield 30
```

I think that will prove that any is short circuiting iteration, or is it not?